config/authentication/group.cfg

```diff
@@ -1,8 +1,10 @@
-admin:admin
-admin:nsadmins
-demo:admin
+admin:scheduleradmins
+admin:rmcoreadmins
+demo:scheduleradmins
+demo:rmcoreadmins
 guest:guests
 nsadmin:nsadmins
+nsadmin2:nsadmins
```

Comment:
Thanks to you, we have now well defined groups: scheduleradmins, rmcoreadmins, nsadmins...  with well defined members: admin, demo... but cant these latter ones replace nsadmin and nsadmins users?

Reply:
@mbenguig nsadmins users are only allowed to control node sources, not the full resource manager